test/run-test.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable no-console */
+
+const { spawn } = require('child_process');
```

Comment:
This seems very similar to the test infrastructure code that @nullvoxpopuli recently added to ember-codemod-telemetry-helpers, maybe we can share?